docs/labels.rst

```diff
@@ -0,0 +1,127 @@
+.. -*- coding: utf-8 -*-
+
+=================================
+Managing Issues and Pull Requests
+=================================
+
+Introduction
+============
+
+We are using the Github issue trackers of the repositories at `github.com/plone/* <https://github.com/plone>`_
+Here our issues for all Plone core repositories are handled.
+
+Pull Requests are issue-like at Github.
+They share the set of labels and milestones.
+
+New issues should be created in `Products.CMFPlone <https://github.com/plone/Products.CMFPlone>`_.
+If the submitter is sure the issue belongs to a specific package, the issue can be created there too.
+
+
+Labels explained
+================
+
+Main Labels
+-----------
+
+We have a well defined set of labels for all Plone core packages and some related packages.
+We tried to find a label system that is not too complex but covers several use cases.
+
+01 type: bug
+    something does not work
+02 type
+    regression missing stuff from former version
+03 type: feature (plip)
+    really new stuff or major change
+04 type: enhancement
+    makes existing code better
+05 type: question
+    discussing
+
+11 prio: blocker
+    need to be fixed immediatly
+12 prio: high
+    High Priority
+13 prio: normal
+    Normal priority
+14 prio: low
+    Low priority
+
+21 status: confirmed
+    issue was read and got confirmed
+22 status: in-progress
+    somebody works on that issue
+23 status: testing
+    pull request specific, a PR test or similar is running, waiting for results
```

Comment:
Do we really need this one? with ``in-progress`` and ``ready`` I would say is enough. As long as jenkins is not happy the issue/pull request is in progress.